contributing/ci-docs.txt

```diff
@@ -370,6 +385,34 @@ The repository for OMERO.figure is https://github.com/ome/omero-figure.
 		   this PR is merged, the GitHub Pages service updates the content of
 		   http://figure.openmicroscopy.org
 
+
+OME Files
+^^^^^^^^^
+
+This bundle of sphinx documentation has two components: OME Files C++
+documentation is located in the ome-files-cpp repository; OME CMake
+Super-Build documentation is located in the ome-cmake-superbuild repository.
+Both are currently built from the master branches despite the build names.
+
+.. glossary::
+
+     :jenkinsjob:`OME-FILES-CPP-DEV-release-bundle-docs`
+
+	    This job is used to publish the master branches of the OME Files C++
+	    and OME CMake Super-Build sphinx documentation as a single bundle
+
+	    #. |buildFilesSB|
+	    #. |deploy-doc| http://www.openmicroscopy.org/site/support/ome-files-cpp/
+
+The merge and latest builds for this documentation set are detailed on the
+:doc:`ci-ome-files` page. The staging location for reviewing documentation PRs
+opened against either of these repositories is
+https://www.openmicroscopy.org/site/support/ome-files-cpp-staging/
+
+Note that because of the bundle nature of these builds, they does not use the
+standard Sphinx configuration described above nor report broken links parsed
```

Comment:
Should be sorted now - grepping reveals only lowercase sphinx occurrences are directory names or within commands